### ISSUE TITLE
ValueError: I/O operation on closed file.

### ISSUE THREAD

**Issue by jyn514** [xmendez/wfuzz#97]
Steps to replicate:

1. install `wfuzz`
2. run `wfuzz -w big.real.txt -w extensions_common.txt <your url>/FUZZFUZ2Z`
3. get the text `ValueError: I/O operation on closed file.`

If you run pdb and follow the traceback far enough, it's coming from `plugins/payloads/file.py:43`: `self.__count = len(self.f.readlines())`

The root cause is actually in the same file in `__next__`:
```
    def __next__(self):
        line = self.f.readline().strip()
        if line == '':
            self.f.close()
            raise StopIteration
        return line
```

Fix: change `self.f.close()` to `self.f.seek(0)`.

**Comment by 3riatarka** [comment]
Works for me, thank you!

**Comment by xmendez** [comment]
Fixed in 2.3.1
